lib/lwt/lwt_xmlHttpRequest.ml

```diff
@@ -170,9 +170,7 @@ let perform_raw
         | `Fields _strings ->
           override_method "POST",
           override_content_type "application/x-www-form-urlencoded"
-        | `FormData _ ->
-          override_method "POST",
-          override_content_type "multipart/form-data"
+        | `FormData _ -> override_method "POST", None
```

Comment:
`None` -> `content_type`. If the user really wants to provide a custom `content_type`, we should respect it. Sorry if I suggested `None` earlier :).